Kit/plugins/Charts.swift

```diff
@@ -327,10 +331,12 @@ public class LineChartView: NSView {
                 path.stroke()
                 
                 let date = self.dateFormatter.string(from: nearest.value.ts)
+                let subtitleWidth = date.widthOfString(usingFont: subtitleFont)
+                let width = max(70, subtitleWidth) + 8
                 let roundedValue = (nearest.value.value * 100).rounded(toPlaces: 2)
                 let strValue = roundedValue >= 1 ? "\(Int(roundedValue))\(suffix)" : "\(roundedValue)\(suffix)"
                 let value = toolTipFunc != nil ? toolTipFunc!(nearest.value) : strValue
-                drawToolTip(self.frame, CGPoint(x: nearest.point.x+4, y: nearest.point.y+4), CGSize(width: 78, height: height), value: value, subtitle: date)
+                drawToolTip(self.frame, CGPoint(x: nearest.point.x+4, y: nearest.point.y+4), CGSize(width: width, height: height), value: value, subtitle: date)
```

Comment:
how this change is related to the date format?

Reply:
it is because not every locale has the same format and so different locales have different width resulting strings. if i did not add this, some locales would have their strings truncated based on the existing width.

Reply:
there was a reason why I do that with static width: with dynamic there was too many resizes and tooltip box "jumps" all the time

Reply:
ha thats why i tried the monospaced font, before you flagged that as not working. once again it would _really_ awesome to figure out what version that is broken in and gracefully fall back, but that is out of the scope of this PR. i just retested this. here is a video in my locale. typically only a single digit is changing. is this acceptable? 

https://github.com/user-attachments/assets/f4eafe7a-6f9a-4ec1-916d-b5ef86f92f21

Reply:
nope, I really do not like jumping boxes. If you want to get more info about monospaced font just search in issues about that, there was a few times when I use that font.

Reply:
given the variance is within a few pixels in the absence of a monospace font. can i just calculate the width for the locale in the constructor add like a few pixels of buffer and then reuse that width in the render to prevent the jitter?